brainstormforce/ultimate-addons-for-gutenberg
# Content Timeline - display post date not working when layout is left/right and heading tag not working issue

### Description

- Display post date not working when layout is left/right and heading tag not working issue
- Trello - https://trello.com/c/LKCRn3Jf/186-content-timeline-issue-of-display-date-in-orientation-left-right

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Content Timeline in master set oriantation left/right.
- Switch to `Content-timeline-date-issue` and is validation issue occured or not.
- Check Content Timeline > Heading Typography is working or not.
- Check Display date in left/right orientation is working or not.

### Checklist:
